estate/models/estate_property_type.py

```diff
@@ -1,8 +1,30 @@
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class EstatePropertyType(models.Model):
     _name = "estate.property.type"
     _description = "Estate Property Type"
+    _order = "sequence, name"
 
+    sequence = fields.Integer(default=1)
     name = fields.Char(required=True)
+    property_ids = fields.One2many(
+        "estate.property",
+        "property_type_id",
+        string="Property Types",
```

Comment:
I'm so sorry, I was thinking that this field was for `estate.property.type` model.

```suggestion
        string="Properties",
```